### ISSUE TITLE
Feature request: Garmin varia radar 515/315 bluetooth support.

### ISSUE THREAD

**Issue by picchiov** [Sublimis/UrbanBiker#105]
Garmin has recently introduced that new bike radar units that now can dialogate with the smartphone via bluetooth.
It would be nice to begin introduced inside the Urban Biker app to improve bikers safety.

**Comment by Sublimis** [comment]
Hi, as much as we would like to implement this, unfortunately for now it doesn't seem possible. The problem is that Garmin never revealed the Bluetooth protocol they use with the Varia radar, and there's no standard Bluetooth "radar" protocol description as you can verify e.g. [here](https://www.bluetooth.com/specifications/gatt/). So the radar only works with their own application, as far as we know.

In other words, we do not know how to connect to the Varia radar. Even if we somehow manage to do that, we do not know what the received data fields mean, so we would have to reverse-engineer. But we won't do that, as it would take too much time, and is probably illegal. :disappointed: Sorry, but thanks for the feedback! If anything changes regarding this, please do let us know!

**Comment by radiotbo** [comment]
Maybe this info will be useful. It will be great to add support for Garmin Varia Radar and light. I will be pleasant to be a beta tester.
[https://forums.garmin.com/developer/connect-iq/f/discussion/240452/bluetooth-profile-for-garmin-varia-rtl515](https://forums.garmin.com/developer/connect-iq/f/discussion/240452/bluetooth-profile-for-garmin-varia-rtl515)
